plutus-core/plutus-core/src/PlutusCore/Crypto/BLS12_381/Pairing.hs

```diff
@@ -49,6 +50,9 @@ instance Flat MlResult where
 instance NFData MlResult where
     rnf _ = ()
 
+instance Hashable MlResult where
+    hashWithSalt salt = const salt
```

Comment:
This is needed to make `Term` hashable, and there doesn't seem to be anything else we can do. @kwxm